codeclimate-govet.go

```diff
@@ -35,12 +50,13 @@ func main() {
 				pieces := strings.Split(line, ":")
 
 				if len(pieces) < 3 {
-					// engine.PrintWarning()
+					fmt.Fprintf(os.Stderr, "Unexpected format for the following output: %v\n", out)
 					return
 				} else {
 					lineNo, err := strconv.Atoi(pieces[1])
 					if err != nil {
-						// engine.PrintWarning()
+						fmt.Fprintf(os.Stderr, "Unexpected format for the following output: %v\n", lineNo)
```

Comment:
I think this should be `line` rather than `lineNo` -- when the format is unexpected, I think we want to print the whole line, not just the line number -- and we may not even have the line number in the case where we failed to coerce it to an integer